app/models/measurement.rb

```diff
@@ -1,5 +1,7 @@
 class Measurement < ActiveRecord::Base
   belongs_to :person
+  scope :descending, -> { order(date: :desc) }
```

Comment:
We could. I myself find that more confusing. (Not in the scope declaration, of course, but when it is called in the view or something.)